### ISSUE TITLE
PresentationNode nodes method creates global

### ISSUE THREAD

**Issue by thecapdan** [BladeRunnerJS/brjs#1219]
```

br.presenter.node.PresentationNode.prototype.nodes = function(sNodeName, vProperties)
{
	sNodeName = (sNodeName && (sNodeName != "*")) ? sNodeName : null;
	mProperties = this._convertToMap(vProperties);
...
```
mProperties should be a var.

Who QAed this stuff?!

![tumblr_m3y9g5fyrr1rv4pd3o1_500](https://cloud.githubusercontent.com/assets/6085260/6225063/3e3755c0-b67d-11e4-82db-c5a69fede6c5.gif)

**Comment by andyberry88** [comment]
> Who QAed this stuff?!

Probably that Dan guy. We had to send him off to another country to get rid of him :smile:

**Comment by janhancic** [comment]
btw @thecapdan you can use a back-tick to format the code here:

```
`this will be formatted`
```

Or if you want to format a block you use 3 back-ticks to start the block, and 3 back-ticks to close it.

**Comment by thecapdan** [comment]
thanks @janhancic . corrected. I usually do, I must have reported that bug at the end of a 14 hour work day;)

**Comment by andyberry88** [comment]
This was fixed by @lukeapage in b8d5591a8726151c365244f4d40cc029de70b3ee.
